advanced_functionality/huggingface_byo_scripts_and_data/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2886 @@
+{
```

Comment:
<p>As a general rule, can you please clean up the cell output? It makes it cleaner for the reviewers, especially if you want to make any changes to this notebook in the future</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>